### PR TITLE
Extract hidden dependency into formal argument.

### PR DESCRIPTION
**Pull request by markstory**
Pull out hidden dependencies into constructor arguments. This makes for less fishing around in what was effectively global variables.

### CODE REVIEW

File: lintreview/processor.py [hunk 1]
```diff
@@ -20,16 +20,15 @@ class Processor(object):
     _config = None
 
     def __init__(self, repository, number, head, target_path, config=None):
+        config = config if config is not None else {}
```
Comment (adrianmoisey) [inline lintreview/processor.py]: What about:
'config = config if config else {}'

**Comment by adrianmoisey** [comment]
Seems legit 🐔

File: lintreview/processor.py (hunk 1 above)
Comment (markstory) [inline lintreview/processor.py]: Good idea.

File: lintreview/review.py [hunk 2]
```diff
@@ -104,11 +101,13 @@ class Review(object):
     to github.
     """
 
-    def __init__(self, gh, number):
+    def __init__(self, gh, number, config=None):
+        config = config if config is not None else {}
```
Comment (adrianmoisey) [inline lintreview/review.py]: Sorry, I forgot to point out that this can also be `config = config if config else {}`.
My bad!

**Pull request merged**